.travis.yml

```diff
@@ -14,15 +14,11 @@ matrix:
 script:
   - npm run lint
   - npm run test
-  - npm run coverage
 
 before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
-after_success:
-  - npm run coverage-publish
-
```

Comment:
Doesn't seem that coverage got added to Circle https://github.com/libp2p/js-libp2p-keychain/blob/master/circle.yml